### PR TITLE
Added support for My Limits view in fornt-end and back-end

### PR DESCRIPTION
**Pull request by MartynasBas**
Added My Limits functionality to front-end and back-end

### CODE REVIEW

File: SchedulearnBackend/SchedulearnBackend/Controllers/LimitController.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            return await _limitService.GetLimitAsync(id);
+        }
+        // GET: api/Limit/user/5
+        [HttpGet("user/{userId}")]
+        public async Task<ActionResult<Limit>> GetLimitByUser(int userId)
+        {
+            System.Diagnostics.Debug.WriteLine($"GetLimitByUser {userId}");
+            return await _limitService.GetLimitByUserAsync(userId);
+        }
+
```
Comment (AinoraZ) [inline SchedulearnBackend/SchedulearnBackend/Controllers/LimitController.cs]: Excesive whitespace

File: schedulearn-frontend/src/App.tsx [hunk 2]
```diff
@@ -44,17 +48,22 @@ export default class App extends React.Component {
               </Route>
               <PrivateRoute path="/list_example">
                 <TopicList />
-                <AnalyzeData />
+                <AnalyzeData />
```
Comment (AinoraZ) [inline schedulearn-frontend/src/App.tsx]: Excesive whitespace

**Review by AinoraZ — CHANGES_REQUESTED** [review]

File: schedulearn-frontend/src/server-components/LimitViewer.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  limitOfLearningDaysPerMonth?: number;
+  limitOfLearningDaysPerQuarter?: number;
+  limitOfLearningDaysPerYear?: number;
+}
+
+export default class LimitViewer extends React.Component<LearningLimitsProps, LearningLimitsState> {
+  state: LearningLimitsState = {}
+  componentDidMount(): void {
+    if (!UserContext.user)
+      throw new Error("Should never reach this calendar when not logged in");
```
Comment (DominykasP) [inline schedulearn-frontend/src/server-components/LimitViewer.tsx]: Fix error message

**Review by DominykasP — CHANGES_REQUESTED** [review]

**Review by DominykasP — APPROVED** [review]
🥇

**Review by AinoraZ — APPROVED** [review]

**Comment by AinoraZ** [comment]
[My limits page](https://trello.com/c/Y755QLZM/4-my-limits-page)

**Pull request merged**